scripts/ci/distcheck.sh

```diff
@@ -14,8 +14,8 @@ cd "$(dirname "$0")"/../..
 # and systems might differ in performance.
 export CI="true"
 
-make distcheck
+make -j $(nproc) distcheck
```

Comment:
That is interesting moment. Travis is passed but wouldn't it bring some suprises in future?  Tests like pktio use veth devices and probably will not work correctly if different tests run in the same time.

Reply:
Then this needs to be fixed. Let's make parallel tests work in the end.

Reply:
I'll remove -j from distcheck as I don't have time to test it more today.